Tests/Marketplace/upload_packs_private.py

```diff
@@ -652,28 +653,22 @@ def print_packs_summary(packs_list):
                      or pack.status == PackStatus.FAILED_DETECTING_MODIFIED_FILES.name]
     failed_packs = [pack for pack in packs_list if pack not in successful_packs and pack not in skipped_packs]
 
-    print("\n")
-    print("------------------------------------------ Packs Upload Summary ------------------------------------------")
-    print(f"Total number of packs: {len(packs_list)}")
-    print("----------------------------------------------------------------------------------------------------------")
-
+    logging.info(
+        f"""------------------------------------------ Packs Upload Summary ------------------------------------------
+        Total number of packs: {len(packs_list)}
+        ----------------------------------------------------------------------------------------------------------""")
```

Comment:
the extra spaces at the beginning of the lines will mess the formatting.

Reply:
![image](https://user-images.githubusercontent.com/41257953/97853818-4e413080-1d01-11eb-909e-d431e43b8a9d.png)